alts/src/main/proto/grpc/gcp/altscontext.proto

```diff
@@ -47,4 +47,5 @@ message AltsContext {
 
   // Additional attributes of the peer.
   map<string, string> peer_attributes = 7;
-}
\ No newline at end of file
+}
+
```

Comment:
No need to add newline at line 51. Just need new line at end of file on line 50.

Reply:
I'm not sure how to interpret this statement.